src/seml/commands/start.py

```diff
@@ -898,7 +898,7 @@ def start_experiments(
         set_to_pending=set_to_pending and local,
     )
 
-    if debug_server:
+    if debug_server and staged_experiments:
```

Comment:
Nice catch! But could we please instead just have a separate check for `staged_experiments` to be non-empty and fail earlier instead of propagating the empty array and failing later?